platform-sdk/swirlds-common/src/main/java/com/swirlds/common/wiring/WiringConfig.java

```diff
@@ -41,11 +41,11 @@
  */
 @ConfigData("platform.wiring")
 public record WiringConfig(
-        @ConfigProperty(defaultValue = "false") boolean healthMonitorEnabled,
-        @ConfigProperty(defaultValue = "true") boolean hardBackpressureEnabled,
-        @ConfigProperty(defaultValue = "1.0") double defaultPoolMultiplier,
-        @ConfigProperty(defaultValue = "0") int defaultPoolConstant,
+        @ConfigProperty(defaultValue = "true") boolean healthMonitorEnabled,
+        @ConfigProperty(defaultValue = "false") boolean hardBackpressureEnabled,
+        @ConfigProperty(defaultValue = "0") double defaultPoolMultiplier,
+        @ConfigProperty(defaultValue = "8") int defaultPoolConstant,
```

Comment:
`PreHandleWorkflowImpl.preHandle()` submits parallel tasks to the current `ForkJoinPool` (i.e. the platform `ForkJoinPool`). Signature verifications are CPU-heavy.  Setting the default parallelism to 8 may be too limiting. 
What's the rationale behind the change?

Reply:
It's what Cody had this set to when he left. The idea was to use these config values as a starting point, and evaluate performance from there.

Reply:
I suggest not changing them (based on my evaluation of current performance).

Reply:
Discussed offline, we have agreement to move forward with the current values.